src/main/main.ts

```diff
@@ -51,7 +52,7 @@ const isDebug =
   process.env.NODE_ENV === 'development' || process.env.DEBUG_PROD === 'true';
 
 if (isDebug) {
-  require('electron-debug')();
+  debug();
```

Comment:
Can you update this with: 

```
if (isDebug) {
  import('electron-debug')
    .then((electronDebug) => electronDebug.default())
    .catch((error) => log.error('Failed to initialize electron-debug:', error));
}
```

This would make sure that we don't include electronDebug in release